defaults/main.yml

```diff
@@ -29,7 +30,7 @@ redis_local_facts: true
 redis_service_name: "redis_{{ redis_port }}"
 
 ## Networking/connection options
-redis_bind: 0.0.0.0
+redis_bind: false
```

Comment:
This might actually be an issue for us if we're relying on implicit device binding

Reply:
We should not be. This default is safer.